### PR TITLE
translations: improve handling of plural forms (resolves #133)

### LINKED ISSUE #133: Improve handling of plural forms
Some messages in the program use two different strings: a "1 item" one, and a "more than 1 item" one, for example:
```
1 file removed
%d files removed
```
While this satisfies the rules of English grammar, it is not enough for many other languages, which have more complicated rules - for example, in Polish, this would be:
```
usunięto 1 plik
usunięto 2-4 pliki
usunięto 5-21 plików
usunięto 22-24 pliki
usunięto 25-31 plików
[...]
```
As can be seen, there are three different forms in use.

Universal support for any language can be achieved using the _plural forms_ feature of gettext.
https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html

### PR DESCRIPTION
**Pull request by andy5995**
After this is merged, 3 plural strings will need to be reviewed for
changes in each po file.

It's not quite clear to me yet though about the subscripted values, so I
need to understand that better before advertising to translators about
these changes.

@sexybiggetje, can I ask you to briefly review this patch before it gets
merged? It might help if there are at least 2 people who are aware of
the subtle change that's being implemented here. Also, if you have any
feedback about it, noting something I missed, please let me know.

Note:There were 3 source code files I changed.

This is the documentation I used to implement the change:
https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html

### CODE REVIEW

**Comment by andy5995** [comment]
>It's not quite clear to me yet though about the subscripted values, 

This is an example, excerpted from a po file:

```
 msgid "%d file purged"
-msgstr "%d bestand gepurged"
+msgid_plural "%d files purged"
+msgstr[0] "%d bestand gepurged"
+msgstr[1] "%d bestand gepurged"
```

**Comment by andy5995** [comment]
These are the changes to the actual source code:

https://github.com/andy5995/rmw/pull/149/files#diff-9169dfe9f2fb8f085e35247729dca23c

https://github.com/andy5995/rmw/pull/149/files#diff-fd962b1925ec098ec73e66c46beb08ca

https://github.com/andy5995/rmw/pull/149/files#diff-903b11d1ac3bee9746b853644e245829

File: po/nl_NL.po [hunk 1]
```diff
@@ -432,19 +432,20 @@ msgid "%d directories skipped (contains read-only files)\n"
...
 
 #: src/purging_rmw.c:392
-#, c-format
+#, fuzzy, c-format
+#| msgid "%d file purged"
 msgid "%d file purged"
-msgstr "%d bestand gepurged"
+msgid_plural "%d files purged"
+msgstr[0] "%d bestand gepurged"
+msgstr[1] "%d bestand gepurged"
```
Comment (sexybiggetje) [inline po/nl_NL.po]: msgstr[1] "%d bestanden gepurged"

**Comment by andy5995** [comment]
Thanks @sexybiggetje and @sapake !

**Pull request merged**